botfront/imports/api/storyGroups/storyGroups.methods.js

```diff
@@ -3,6 +3,28 @@ import { check } from 'meteor/check';
 
 import { StoryGroups } from './storyGroups.collection';
 
+export const createIntroStoryGroup = (projectId) => {
+    if (!Meteor.isServer) throw Meteor.Error(401, 'Not Authorized');
+    Meteor.call(
+        'storyGroups.insert',
+        {
+            name: 'Intro Story Group',
```

Comment:
Please rename Intro Story Group to "Intro stories"

Reply:
Done